developer-docs-site/docs/concepts/basics-accounts.md

```diff
@@ -2,42 +2,116 @@
 title: "Accounts"
 id: "basics-accounts"
 ---
-An account represents a resource on the Aptos Blockchain that can send transactions. Each account is identified by a particular 32-byte account address and is a container for Move modules and Move resources.
 
-# Introduction
+# Accounts
 
-The state of each account comprises both code and data:
+An account on the Aptos Blockchain contains blockchain assets. These assets, for example, coins and NFTs, are by nature scarce and must be access-controlled. Any such asset is represented in the blockchain account as a **resource**. A resource is a Move language primitive that emphasizes access control and scarcity in its representation. However, a resource can be used to represent entities other than assets.
```

Comment:
other than assets without a qualification is ... a bit weird. I guess you could say other on-chain capabilities, identifying information, and access control.